npm/private/npm_translate_lock_helpers.bzl

```diff
@@ -426,15 +426,16 @@ ERROR: can not apply both `pnpm.patchedDependencies` and `npm_translate_lock(pat
                 # in pnpm-lock.yaml which we must replace with the desired registry in the `registry` field:
                 #   tarball: https://registry.npmjs.org/@types/cacheable-request/-/cacheable-request-6.0.2.tgz
                 #   registry: https://registry.yarnpkg.com/
-                if registry and tarball.startswith(utils.default_registry()):
-                    url = registry + tarball[len(utils.default_registry()):]
+                if registry and tarball.startswith(default_registry):
```

Comment:
This should have no change since `utils.default_registry()` is passed into this method already, but we should use the param instead of invoking that util again.